Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```
@@ -25,6 +25,7 @@
       - id: AstroNavCartridge
       - id: LunchboxCommandFilledRandom # Delta-V Lunchboxes!
         prob: 0.3
+      - id: PrinterDocFlatpack # Corvax-Printer
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)